display/ssd1306.h

```diff
@@ -450,14 +468,14 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
   void SB_Effect(EffectType effect, float location) override {
     switch (effect) {
       case EFFECT_BOOT:
-	if (IMG_boot) {
-	  ShowFile(&IMG_boot, font_config.ProffieOSFontImageDuration);
+      	if (IMG_boot) {
+	  ShowFile(&IMG_boot, font_config.ProffieOSBootImageDuration);
```

Comment:
For backwards compatibility, I think this needs to default to ProffieOSFontImageDuration if ProffieOSBootImageDuration isn't set.

Reply:
Not sure I understand this one.
Boot and font were the original 2 images supported.
And BootImage has a default set in hybrid_font incase not set in the font config.ini.